Sources/DolbyIORTSUIKit/Private/Views/StatsInfo/StatsInfoViewModel.swift

```diff
@@ -5,11 +5,95 @@
 import Combine
 import DolbyIORTSCore
 import Foundation
+import SwiftUI
```

Comment:
Let's not use SwiftUI in the ViewModel's ? For localisation of key's used in the ViewModels we can probably use String(localized: 'key-name', bundle: .module) initialiser instead ? Also in the View's we can use the Text(verbatim: font:) for the text component.